scheme/proto/src/test/proto/scheme.proto

```diff
@@ -66,5 +70,9 @@ message ValueSchemeMessage {
   bool boolean_type = 5;
   uint64 long_type = 6;
   uint32 int_type = 7;
+  repeated bytes repeated_bytes = 8;
+}
 
+message RecursiveMessage {
+  RecursiveMessage recursion = 1;
```

Comment:
It would be cool to add additional set of messages, that will test changing the wire-compatible changes (repeated <-> optional) and verify that everything works as expected. Protobufs are very likely to evolve and what is allowed to do in protobuf should be allowed in the schema as well.